canvas_sdk/v1/data/message.py

```diff
@@ -0,0 +1,79 @@
+from django.db import models
+
+
+class TransmissionChannel(models.TextChoices):
+    """Transmission channel."""
+
+    MANUAL = "manual", "Manual"
+    TEXT_MESSAGE = "sms", "Text Message"
+    EMAIL = "email", "Email"
+    NOOP = "noop", "No-op"
+
+
+class Message(models.Model):
+    """Message."""
+
+    class Meta:
+        managed = False
+        db_table = "canvas_sdk_data_api_message_001"
+
+    id = models.UUIDField()
+    dbid = models.BigIntegerField(primary_key=True)
+    created = models.DateTimeField()
+    modified = models.DateTimeField()
+    content = models.TextField()
+    sender = models.ForeignKey(
+        "v1.CanvasUser", on_delete=models.DO_NOTHING, related_name="sent_messages", null=True
+    )
+    recipient = models.ForeignKey(
+        "v1.CanvasUser", on_delete=models.DO_NOTHING, related_name="received_messages", null=True
+    )
+    note = models.ForeignKey(
+        "v1.Note", on_delete=models.DO_NOTHING, related_name="message", null=True
+    )
+    read = models.BooleanField()
+
+
+class MessageAttachment(models.Model):
```

Comment:
So awesome we will see the attachments that the patient sends to staff. But does the create message allow for attaching a file? What if the staff wants to send a file

Reply:
Nope, although I've added this to the model, the effect doesn't yet support it, sorry!

Reply:
It will definitely be a future enhancement I think some customers will push for but this is getting us in parity with what our product currently does